src/tests/ftest/api/simple.py

```diff
@@ -0,0 +1,102 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+
+from apricot import TestWithServers
+from cmocka_utils import CmockaUtils
+from job_manager_utils import get_job_manager
+
+
+class Simple(TestWithServers):
```

Comment:
I know it's redundant, but can we rename this to `APISimple`? Class names are unique across all functional tests, so `Simple` isn't clear with respect to other areas.